### ISSUE TITLE
manage.py makemigrations: error: unrecognized arguments: --initialize-max-migrations

### ISSUE THREAD

**Issue by mrswats** [adamchainz/django-linear-migrations#11]
Hello! 

Just saw this package from the django-mailing-list and I immediately knew I needed this as as early as today we had this issue.

So, I installed it in our repository and when running the command in the README I am getting the following error:

```
$ ./manage.py makemigrations --create-max-migration-files
SystemCheckError: System check identified some issues:

ERRORS:
?: (dlm.E001) tracker's max_migration.txt does not exist.
        HINT: If you just installed django-linear-migrations, run 'python manage.py makemigrations --initialize-max-migrations'. Otherwise, check how it has gone missing.

```
Fine, but when I run the other command It says the argument is unrecognized.

I am a bit lost, here. Am I doing something wrong? Did I miss anything?

Thanks for the help in advance!

**Comment by adamchainz** [comment]
Ah yes - two problems here. First I didn't update the message from 'initialize' to 'create' when I decided to rename it during development. Second I hadn't tested that part on a real project, only in unit tests, and there Django disables the system checks - so I hadn't seen that you can't run the command since the check blocks it.

Fixed in #12, which has been released in version 1.0.1. There's now a separate command to run at the start, `manage.py create-max-migration-files`.

**Comment by mrswats** [comment]
Thanks for the quick turnaround! Will try it now, again! :)
